eden/scm/edenscm/testing/ext/hg.py

```diff
@@ -51,7 +51,7 @@ def testsetup(t: TestTmp):
             if setup:
                 testname = os.path.basename(testfile)
                 setup(testname, str(hgrcpath))
-    inprocesshg = True
```

Comment:
This would make tests run much slower. We should only use it in incompatible tests.